controllers/api.js

```diff
@@ -15,6 +15,12 @@ redis.monitor(function (err, monitor) {
   });
 });
 
+/****** START SET FEES FROM CONFIG AT STARTUP ******/
+/** GLOBALS */
+global.forwardFee = config.defaultForwardReserveFee;
+global.internalFee = config.defaultIntraHubFee;
```

Comment:
Is there a way to pass these settings into Paym instead of relying on globals?

Reply:
Please try and report back ...